src/test/java/gov/healthit/chpl/aqa/pageObjects/CollectionsPages.java

```diff
@@ -41,4 +41,12 @@ public static WebElement cmsFaqLink(final WebDriver driver) {
         return element;
     }
 
+    /**
+     * Returns element holding SED Details File.
+     * @param driver WebDriver
```

Comment:
In the other CHPL projects we're moving away from requiring javadocs on super simple methods like this one. If you want to do the same thing you can add `<property name="minLineCount" value="10" />` to the Checker -> TreeWalker -> JavadocMethod element of the Checkstyle file